client/src/components/ApplicationDashBoard/ApplicationDashBoard.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import { useAppSelector } from "../../app/hooks";
+
+interface IStatusCount {
+  status: string;
+  count: number;
+}
+
+const ApplicationDashboard = (): JSX.Element => {
+  const [totalApplications, setTotalApplications] = useState(0);
+  const [applicationsByStatus, setApplicationsByStatus] = useState<
+    IStatusCount[]
+  >([]);
+  const user = useAppSelector((state) => state.user.userData);
+
+  useEffect(() => {
+    async function fetchAggregatedData() {
+      try {
+        const response = await axios.get(
+          `/api/applications/aggregated-user-stats?user_id=${user?._id}`
+        );
+        setTotalApplications(response.data.totalApplications);
+        setApplicationsByStatus(response.data.applicationsByStatus);
+      } catch (error) {
+        console.error("Error fetching aggregated data:", error);
```

Comment:
Is there a ticket for creating some sort of generic error boundary component? Would be a good medium difficulty task!

Reply:
@JimmyTran48 I have a ticket up for a user feedback modal that would handle success and error messaging.